### PR TITLE
fix(core): Deactivate active workflows during import

### PR DESCRIPTION
**Pull request by flipswitchingmonkey**
Github issue / Community forum post (link here to close automatically):
https://community.n8n.io/t/webhooks-does-not-register-when-importing-workflows-via-the-n8n-cli-command/22330

This will deactivate workflows imported using the import:workflow command, since the CLI command only imports a workflow into the DB, but is a separate process in itself and can not activate the webhook in another process (the running n8n instance).

This also prevents immediate activation of imported but untested workflows.

### CODE REVIEW

**Review by krynble — CHANGES_REQUESTED** [review]
We also need to do the same at around line 185 or maybe make the change in the `storeWorkflow` function.

**Review by krynble — CHANGES_REQUESTED** [review]
I think we can remove the backup file. All the rest looks fine! Thanks!

File: packages/cli/backup/out.json
[the remark this answers is not in the document]
Comment (flipswitchingmonkey) [reply packages/cli/backup/out.json:1]: whoops, no that made it into the commit unasked...

**Review by krynble — APPROVED** [review]
LGTM!

**Pull request merged**